mne/io/pick.py

```diff
@@ -786,6 +791,17 @@ def _mag_grad_dependent(info):
                for ph in info.get('proc_history', []))
 
 
+def _fnirs_raw_dep(ch_type, warned):
+    if ch_type == 'fnirs_raw':  # alias
+        if not warned[0]:
+            warn('"fnirs_raw" has been deprecated in favor of the more '
+                 'explicit "fnirs_cw_amplitude" and will be removed in 0.22',
+                 DeprecationWarning)
```

Comment:
Just a question, how do you go about ensuring these things are removed as specified by the depreciation warnings? Do you just do a search for `DeprecationWarning` after releasing 0.21?

Reply:
Yes, it's my favorite part of the [release process (step 14)](https://github.com/mne-tools/mne-python/wiki/How-to-make-a-release)...

Reply:
Thanks for the link. I'm just doing some prethinking for when we eventually push MNE-NIRS out (if we do).

Reply:
@rob-luke what do you mean by MNE-NIRS?

Reply:
https://mne.tools/mne-nirs/

Reply:
Its where I do the experimental stuff, get everything working, tweak the API, then move to MNE-Python

Reply:
Got it - I'm glad all of the NIRS functionality will eventually land in MNE-Python.

Reply:
Thats the plan. Definitely the MNE epoching style stuff goes straight in here. But the different style analysis (more MRI than EEG) will start of in MNE-NIRS.